types/react-window-infinite-loader/index.d.ts

```diff
@@ -14,6 +14,8 @@ interface InfiniteLoaderProps {
     loadMoreItems: (startIndex: number, stopIndex: number) => Promise<any>;
     itemCount: number;
     children: (props: {onItemsRendered: OnItemsRendered, ref: Ref<any>}) => ReactNode;
+    minimumBatchSize: number;
+    threshold: number;
```

Comment:
This prop should be marked as optional.
```suggestion
    threshold?: number;
```